src/en/controllers-creating.md

```diff
@@ -7,20 +7,34 @@ TODO: Review again soon (created: March 2016)
 Use the `juju bootstrap` command to create a controller (and model) for a given
 cloud provider:
 
-`juju bootstrap <controller name> <provider name> [--options]`
+`juju bootstrap [options] <controller name> <provider name>`
 
 See `juju help bootstrap` for details on this command or see the
 [command reference page](./commands.html#juju-bootstrap).
 
 
+## Notes
+
+The '--upload-tools' option is used to make agent software available that is
+more recent than the default binary. This is done when some features may not
+yet be compiled in to the agent for the Ubuntu release being installed. Note
+that Juju will default to the latest LTS (see `distro-info --lts` command).
+
+
```

Comment:
are you using the evil 'upload-tools' in the examples because it is useful for beta/pre-release? I would do the examples as if they were for proper release, and add a note to suggest using upload-tools in these circumstances, otherwise we just have to come back and change examples rather than just removing or rewording a note